packages/app/src/systems/SignUp/pages/RecoverWallet/RecoverWallet.tsx

```diff
@@ -0,0 +1,40 @@
+import { useNavigate } from "react-router-dom";
+
+import {
+  CreatePassword,
+  MnemonicWrite,
+  SignUpFailed,
+  WalletCreated,
+} from "../../components";
+import { useSignUp } from "../../hooks";
+import { SignUpType } from "../../machines/signUpMachine";
+
+import { Layout } from "~/systems/Core";
+
+export function RecoverWallet() {
+  const { state, handlers, context } = useSignUp(SignUpType.recover);
+  const navigate = useNavigate();
+
+  return (
+    <Layout title="Recovering Wallet" isPublic>
```

Comment:
For now, as we don't have a secure store of the seed phrase would be great to add a temporary disclaimers box using warn. "This wallet is current on development, and your phrase is not safely stored, DO NOT IMPORT YOUR CURRENT SEED PHRASE."

Reply:
✅